File: code/1208/1208翻硬币.cpp
#include <iostream>
#include <cstring>
#include <algorithm>
using namespace std;

void change(char &ch) {
	if (ch == '*') ch = 'o';
	else ch = '*';
}

int main() {
	string start, end;
	cin >> start >> end;

	int res = 0;
	for (int i = 0; i + 1 < start.size(); i++) {
		if (start[i] != end[i]) {
			res++;
			change(start[i]);
			change(start[i + 1]);
		}
	}

	cout << res << endl;
		
	return 0;
}

File: code/1208/1208翻硬币.py
if __name__ == "__main__":
    start = list(input())
    end = list(input())

    res = 0
    for i in range(len(start) - 1):
        if start[i] != end[i]:
            res += 1
            start[i] = '*' if start[i] == 'o' else 'o'
            start[i + 1] = '*' if start[i + 1] == 'o' else 'o'
    print(res)
